File: core/forge/src/main/java/com/possible_triangle/multikulti/forge/platform/service/ForgePlatformHelper.java
package com.possible_triangle.multikulti.forge.platform.service;

import com.possible_triangle.multikulti.platform.service.IPlatformHelper;
import net.minecraftforge.fml.ModList;
import net.minecraftforge.fml.loading.FMLLoader;

public class ForgePlatformHelper implements IPlatformHelper {

    @Override
    public boolean isDev() {
        return !FMLLoader.isProduction();
    }

    @Override
    public boolean isModLoaded(String modId) {
        return ModList.get().isLoaded(modId);
    }

}

File: datagen/forge/src/main/kotlin/com/possible_triangle/multikulti/forge/platform/service/ForgePlatformHelper.kt
package com.possible_triangle.multikulti.forge.platform.service

import com.possible_triangle.multikulti.platform.service.IPlatformHelper
import net.minecraftforge.fml.ModList
import net.minecraftforge.fml.loading.FMLLoader

class ForgePlatformHelper : IPlatformHelper {

    override fun isDev(): Boolean {
        return !FMLLoader.isProduction()
    }

    override fun isModLoaded(modId: String): Boolean {
        return ModList.get().isLoaded(modId)
    }

}